index.html

```diff
@@ -39,24 +39,170 @@
 	  </header>
 
       <section class="head flex">
-        <h1 class="head__title fz-40 fw-700 lh-52">I'm Juhar Glad to see you!</h1>
+        <h1 class="head__title fz-40 fw-700 lh-52">I'm Juhar glad to see you!</h1>
         <p class="head__dsc fz-16 fw-400 lh-24">
           I’m a software developer! I can help you build a product , feature or
           website Look through some of my work and experience! If you like what
           you see and have a project you need coded, don’t hestiate to contact
           me.
         </p>
-		<a href="#" class="head__link fz-16 fw-500 lh-24">My Projects</a>
+		<a href="#" class="head__link link fz-16 fw-500 lh-24 clr-p-400">My Projects</a>
 		<div class="head__icons">
 			<ul class="head__icons__ul ul flex">
-				<li class="head__icons__li"><a href="#" class="head__icons__link link"><img src="Image/twitter.png" alt="twitter"></a></li>
-				<li class="head__icons__li"><a href="#" class="head__icons__link link"><img src="Image/Linkedin.svg" alt="Linkedin"></a></li>
+				<li class="head__icons__li"><a href="#" class="head__icons__link link"><img src="Image/twitter.png" alt="twitter icon"></a></li>
+				<li class="head__icons__li"><a href="#" class="head__icons__link link"><img src="Image/Linkedin.svg" alt="linkedin icon"></a></li>
 				<li class="head__icons__li"><a href="#" class="head__icons__link link"><img src="Image/M.png" alt="M icon"></a></li>
 				<li class="head__icons__li"><a href="#" class="head__icons__link link"><img src="Image/github.svg" alt="github icon"></a></li>
 				<li class="head__icons__li"><a href="#" class="head__icons__link link"><img src="Image/hand.svg" alt="hand icon"></a></li>
 			</ul>
 		</div>
       </section>
+
+	  <section class="works">
+		  <ul class="works__cards ul grid">
+		  <li class="works__card flex bg-white">
+			  <div class="works__card__img">
+				  <img src="Image/tonic-project.png" alt="tonic project">
+			  </div>
+			  <div class="works__card__text">
+				  <h2 class="works__card__title fw-700 fz-32 lh-44 clr-n-800">Tonic</h2>
+				  <div class="works__card__subtitle flex">
+					  <p class="works__card__subtitle-1 fw-600 fz-13 lh-16 clr-n-600">CANOPY</p>
+					  <ul class="works__card__subtitle-ul flex">
+						  <li class="works__card__subtitle-li fw-600 fz-13 lh-16 clr-n-500">Back End Dev</li>
+						  <li class="works__card__subtitle-li fw-600 fz-13 lh-16 clr-n-500">2015</li>
+					  </ul>
+				  </div>
+				  <p class="works__card__dsc fw-400 fz-15 lh-24 clr-n-600">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+				  <div class="works__card__topics flex">
+					  <p class="works__card__topics__text fw-500 fz-12 lh-16 clr-p-400 bg-p-50">html</p>
+					  <p class="works__card__topics__text fw-500 fz-12 lh-16 clr-p-400 bg-p-50">css</p>
+					  <p class="works__card__topics__text fw-500 fz-12 lh-16 clr-p-400 bg-p-50">javascript</p>
+				  </div>
```

Comment:
- [ ] The list of project technologies are a list `ul` and each of them a `li`, please kindly fix it 😉 😉

Reply:
ok thanks, I will change it ASP.